package.json

```diff
@@ -12,14 +12,13 @@
     "format": "prettier . --write --cache --cache-location=node_modules/.cache/.prettiercache --log-level=warn",
     "format:check": "prettier . --check --cache --cache-location=node_modules/.cache/.prettiercache",
     "lint": "turbo run lint",
-    "publint": "turbo run publint --concurrency=1",
```

Comment:
publint is run as part of tsdown when installed.

Reply:
ok